src/styles/components/code/styles.less

```diff
@@ -38,6 +38,11 @@
           }
         }
       }
+
+      .code-copy-icon.tight {
```

Comment:
Let's please not introduce new modifiers like `tight`. Let's reuse what we have from CNVS.

Reply:
I totally agree, but there is no other implementation or generic rule to do this as globally this image is attached to some default rules which is wrong!